docs/05. module guide/11. generate_thumbnails_based_on_folder.md

```diff
@@ -17,3 +17,9 @@ $this->frm->getField('image')->generateThumbnails($imagePath, time() . '.png');
 ```
 
 Where `$imagePath` is the folder that contains the thumbnail folders, if it doesn't exists a folder called source will be created to store the source-file. 
+
+## Generating thumbnails using the CLI
+
+```bash
+app/console thumbnails:generate --folder=backend_users/avatars
```

Comment:
In the locale documentation there is also mentioned "locale:import" and not "forkcms:locale:import"